dist/configs/itwinjs-recommended.js

```diff
@@ -2,27 +2,54 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-/** @type{import("eslint").Linter.BaseConfig} */
-module.exports = {
-  env: {
-    "browser": true
+
+
+const typescriptParser = require("@typescript-eslint/parser");
+
+const typescriptEslintPlugin = require("@typescript-eslint/eslint-plugin");
+const importPlugin = require("eslint-plugin-import");
+const preferArrowPlugin = require("eslint-plugin-prefer-arrow");
+const deprecationPlugin = require("eslint-plugin-deprecation");
+
+const importSpacingRule = require("../rules/import-spacing");
+const importWithinRule = require("../rules/import-within-package");
+const preferGetRule = require("../rules/prefer-get");
+const requireBasicRpcValueRule = require("../rules/require-basic-rpc-values");
+const requireVersionInDeprecationRule = require("../rules/require-version-in-deprecation");
+const noInternalBarrelRule = require("../rules/no-internal-barrel-imports");
+
+module.exports =
+{
+  languageOptions: {
+    ecmaVersion: "latest",
+    sourceType: "module",
+    parser: typescriptParser,
+    parserOptions: {
+      project: "tsconfig.json",
+      ecmaFeatures: {
+        jsx: true,
+      },
+    },
   },
-  extends: [
-    "plugin:@typescript-eslint/recommended",
-    "plugin:@typescript-eslint/recommended-requiring-type-checking",
-  ],
-  parser: "@typescript-eslint/parser",
-  parserOptions: {
-    "project": "tsconfig.json",
-    "sourceType": "module"
+  plugins: {
+    "@typescript-eslint": typescriptEslintPlugin,
+    "import": importPlugin,
+    "prefer-arrow": preferArrowPlugin,
+    "deprecation": deprecationPlugin,
+    custom: {
```

Comment:
instead of building a "custom" plugin here, shouldn't this import the same plugin module as consumers? (like in the readme)

Reply:
I'm not sure what the standard way of doing it is. I'm a little worried that if the plugin is defined twice, you can't override and disable our rules because they will be applied twice.

Reply:
changed